packages/kbn-ebt-tools/src/metric_events/helpers.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { AnalyticsClient } from '@kbn/analytics-client';
+import { type MetricEvent, METRIC_EVENT_SCHEMA } from './schema';
+
+const PERFORMANCE_METRIC_EVENT_TYPE = 'performance_metric';
+
+/**
+ * Register the `metrics` event type
```

Comment:
Nit: metrics -> performance_metric. IMHO we should probably not mention the name here anyway because it's an implementation detail.